core/include/keyman/keyman_core_api_actions.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * Keyman Keyboard Processor API - Internal Action Interfaces
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <keyman/keyman_core_api_bits.h>
+#include <keyman/keyman_core_api_vkeys.h>
+
+#if defined(__cplusplus)
+extern "C"
+{
+#endif
+
+// ----------------------------------------------------------------------------------
+// Actions APIs are now available only to the keyboard debugger, IMX, and Core unit
+// tests (17.0)
```

Comment:
Should we enforce this by putting the definitions inside of an `#ifdef` block and add that define to debugger, IMX and core? Otherwise someone might discover this header file and include it without noticing the disclaimer. Of course it doesn't prevent someone using this but it requires an additional change which hopefully causes the disclaimer to be noticed.

Reply:
No, I think we should just put it into the v17 documentation for Core (TBD!)